packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/shortcut.test.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {getControllerSpec, requestBody, post} from '../../../../../';
+
+describe('requestBody decorator - shortcuts', () => {
+  context('array', () => {
+    it('generates the correct schema spec for an array argument', () => {
+      const description = 'an array of names';
+      class MyController {
+        @post('/greeting')
+        greet(
+          @requestBody.array(
+            {type: 'string'},
```

Comment:
We should allow the parameters to take in primitive wrapper functions. This is so that users can do `@requestBody.array(String, optSpec)` instead of `@requestBody.array({type: 'string'}, optSpec)`. We can leverage `getSchemaForParam` to do this

Reply:
@shimks I didn't implement shortcut functions for `@requestBody.array` and also `@param.array` is because I have different plans for them, and I think it takes discussion time, created an issue for it:

https://github.com/strongloop/loopback-next/issues/1064